kumascript/index.ts

```diff
@@ -37,7 +37,7 @@ export async function render(
     selective_mode = false,
     invalidateCache = false,
   }: RenderOptions = {},
-  doc?: Doc
+  blogPost?: BlogPostDoc
```

Comment:
This doesn't look right. Why would `kumascript.render()` return a `BlogPostDoc`?

Reply:
It doesn't, this is the last argument, the return argument is the line below (the Promise)

Reply:
That should stay called `doc` it just replaces the `Document.findByURL`. And the type is tricky. Probably `Doc` as `BlogPostDoc` should just extend `Doc` but that is utterly broken right now.

Reply:
I don't think `Doc` applies here.

BlogPostDoc:

https://github.com/mdn/yari/blob/006eb85729f34721eea92e5f2146f83dd5c37438/build/blog.ts#L347-L355

DocMetadata/Doc:

https://github.com/mdn/yari/blob/006eb85729f34721eea92e5f2146f83dd5c37438/libs/types/document.ts#L128-L159